src/Pages/BookDetails/Aside.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+function Aside() {
+  const [save, setSave] = useState(false);
+
+  return (
+    <AsideContainer>
+      <Container>
+        <ul>
+          <Menu save={save} onClick={() => setSave(!save)}>
+            <img
+              alt=""
+              src={
+                save
+                  ? "https://i.ibb.co/wSDskYv/shhhh.png"
+                  : "https://cdn4.iconfinder.com/data/icons/app-custom-ui-1/48/Download-256.png"
```

Comment:
```suggestion
                isSaved ? COMPLETED_BTN_IMG : SAVE_BTN_IMG
```
주소를 별도의 변수로 저장해두고 사용한다면 가독성 측면에서 더욱 개선될 수 있을 것 같습니다

Reply:
오 그렇게 해보겠습니다 !